src/fuzzing/ParseMoney/dict

```diff
@@ -0,0 +1,5 @@
+kw1="."
+kw2=" "
+kw3="1"
+kw4="9"
+kw5="\0"
```

Comment:
I think it's the case that having a dictionary of single bytes is less efficient than having no dictionary, at least it was with AFL as it would always try single byte values and also the dictionary value, so it would try each byte in the dictionary in each location twice. We could compare that though by doing a version of this without this file and trying again later. This is too small of an issue to be necessary to fix in this PR though.